mergebot-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Could you please provide some context on how to configure mergebot? Please also add this to the README/CONTRIBUTE doc.